testsuite/src/it/java/io/quarkus/ts/startstop/SpecialCharsTest.java

```diff
@@ -189,6 +191,7 @@ public void specialDEV(TestInfo testInfo) throws IOException, InterruptedExcepti
     @Test
     @Tag("native")
     @DisabledOnOs({OS.WINDOWS}) // https://github.com/quarkusio/quarkus/issues/9707
+    @Disabled("TODO: https://issues.redhat.com/browse/QUARKUS-4695")
```

Comment:
https://github.com/quarkusio/quarkus/issues/42248 should be referenced instead of product tracker